src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -536,17 +536,18 @@ public function pop(): bool {
 	 *
 	 * @see WP_HTML_Open_Elements::pop
 	 *
-	 * @param string $tag_name Name of tag that needs to be popped off of the stack of open elements.
+	 * @param string $tag_name      Name of tag that needs to be popped off of the stack of open elements.
+	 * @param string $tag_namespace Optional. Tag namespace. One of 'html', 'svg', or 'math'. Default 'html'.
 	 * @return bool Whether a tag of the given name was found and popped off of the stack of open elements.
 	 */
-	public function pop_until( string $tag_name ): bool {
+	public function pop_until( string $tag_name, string $tag_namespace = 'html' ): bool {
```

Comment:
when I started updating these methods before, I realized they are only called from the HTML namespace, so instead of adding method arguments, we were able to make the method verify `html` only. I think that could be valuable here too.

Reply:
I noticed that as well when I went to add namespace parameters as necessary at call sites.

I'll update this to use `html` namespace and document that.